### PR TITLE
Bugfix 76  Pie chart labels

### LINKED ISSUE #76: Pie chart labels bug

### PR DESCRIPTION
**Pull request by DZotoff**
#76 Looking for the possible solution for labelLine feature, which is connect the label and the Pie.

### CODE REVIEW

**Review by yuusufisse — APPROVED** [review]
looks good

File: src/components/charts/timebank-piechart.tsx [hunk 1]
```diff
@@ -52,4 +71,5 @@ const TimebankPieChart = ({ personDailyEntry }: Props) => {
   );
 };
 
+
```
Comment (Jdallos) [inline src/components/charts/timebank-piechart.tsx:74]: No need for a double line break here

File: src/components/charts/timebank-piechart.tsx [hunk 2]
```diff
@@ -40,8 +40,16 @@ const TimebankPieChart = ({ personDailyEntry }: Props) => {
           cx="50%"
           cy="50%"
           outerRadius={50}
-          label={renderCustomizedLabel}
-        >
+          label={(entry) => {
+            if (!entry.dataKey) {
+              return null; // do not render any label if dataKey is 0 or empty
```
Comment (Jdallos) [inline src/components/charts/timebank-piechart.tsx:45]: Although sometimes its good to leave comments in the code if what is happening is unclear, I would say in this case these comments are not needed and can be removed.

File: src/components/charts/timebank-piechart.tsx [hunk 3]
```diff
@@ -40,8 +40,16 @@ const TimebankPieChart = ({ personDailyEntry }: Props) => {
...
+            } else {
+              // rendering for non-empty dataKey
```
Comment (Jdallos) [inline src/components/charts/timebank-piechart.tsx:47]: Same here

**Review by Jdallos — APPROVED** [review]

**Pull request merged**